BeVolt/App/src/main.c

```diff
@@ -16,6 +18,7 @@
 #include "SysTick.h"
 #include "PLL.h"
 #include "CLI.h"
+#include "CAN.h"
 
 cell_asic Minions[NUM_MINIONS];
 bool override = false;		// This will be changed by user via CLI
```

Comment:
Can you add a call to DashMessage() inside of the while loop in realmain() so it is called when the BPS is running during the race?